packages/ia-components/components/audio-players/archive-audio-jwplayer-wrapper.jsx

```diff
@@ -133,13 +133,17 @@ class ArchiveAudioPlayer extends Component {
       return;
     }
 
+    const playerStatus = jwplayerInstance.getState();
+    const incomingTrackChange = (incomingTrackNum > prevIndex) || (trackNumber !== incomingTrackNum);
+    const autoplaying = incomingTrackChange && (playerStatus === 'idle');
+
     const iaPlayerisReady = playlistLoadCount === this.maxPlaylistLoadsUntilPlayerIsReady;
     if (!playerReady && iaPlayerisReady) {
       /** IA player starting, let's tell main app where we are starting */
       this.updateAndPlayTrack({
         playerReady: true,
         trackNumber: incomingTrackNum,
-      });
+      }, !autoplaying);
```

Comment:
`this.updateAndPlayTrack` will call `playN` unless you tell it otherwise (2nd arg, bool).  only call playN if it isn't autoplaying